File: apps/api/src/lib/qdrant.ts
import { QdrantClient } from '@qdrant/js-client-rest'

export const qdrant = new QdrantClient({
  url: process.env.QDRANT_URL || 'http://localhost:6333',
  apiKey: process.env.QDRANT_API_KEY || undefined,
})

export const COLLECTION = process.env.QDRANT_COLLECTION || 'gita_shlokas'

File: apps/api/app/lib/qdrant.py
from qdrant_client import AsyncQdrantClient

from app.config import settings


COLLECTION = settings.qdrant_collection


def get_qdrant_client() -> AsyncQdrantClient:
    return AsyncQdrantClient(
        url=settings.qdrant_url,
        api_key=settings.qdrant_api_key,
        check_compatibility=False,
    )
